Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM openshift/origin-release:golang-1.11 as builder
+FROM golang:alpine as builder
 
-# Add everything
 ADD . /usr/src/sriov-cni
 
+ENV HTTP_PROXY $http_proxy
+ENV HTTPS_PROXY $https_proxy
 RUN cd /usr/src/sriov-cni && make build
```

Comment:
Actually this resulted in following error:
```
/bin/sh: make: not found
The command '/bin/sh -c cd /usr/src/sriov-cni && make build' returned a non-zero code: 127
```

I'm working on fix + updating the entrypoint script too to work with alpine base image.